src/components/Apps/Apps.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Breadcrumb } from 'react-breadcrumbs';
+import { connect } from 'react-redux';
+import { Route, Switch } from 'react-router-dom';
+import { AppsRoutes } from 'shared/constants/routes';
+
+class Apps extends React.Component {
```

Comment:
Not sure what your plans are for this file, but does it have to be JavaScript?

Reply:
We can make it typescript, but it'd be the first of these routing components I think.

Wasn't sure if I should break the mold already.

Reply:
Let's do it! 👍🏻